### PR TITLE
Adds Italian translation

### PR DESCRIPTION
**Pull request by verginer**

### CODE REVIEW

File: ita.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "loadButtonLabel": "Clicca per\nCaricare\nPanorama",
+    "loadingLabel": "Caricamento in corso ...",
+	"bylineLabel": "per %s",    
+    "noPanoramaError": "Nessuna immagine panoramica specificata.",
+    "fileAccessError": "Non è possibile elaborare il file %s.",
+    "malformedURLError": "C'è un problema con l'URL del panorama.",
+    "iOS8WebGLError": "WebGL ha degli errori noti su iOS 8, per questo motivo è solo possibile elaborare JPEG con encoding progressivo. (questo panorama usa un encoding standard).",
+    "genericWebGLError": "Il tuo browser non dispone del supporto necessario per WebGL per mostrare questo programma.",
+    "textureSizeError": "Questo panorama è troppo grande per il tuo dispositivo! Ha una larghezza di %spx, ma il tuo dispositivo può solo elaborare immagini con larghezza massima di %spx. Riprova con altro dispositivo. (Se sei l'autore, prova a ridimensionare l'immagine.)",
+    "unknownError": "Errore sconosciuto. Controlla la 'Developer Console'."
```
Comment (DanielBiegler) [inline ita.json]: Is there no word for developer console? Google told me `console di sviluppo`. Or does that sound bad in italian?

Comment (verginer) [reply ita.json]: Actually that is a good translation. I personally refer to it as the "Developer Console" but looking around I have seen "console di sviluppo" being [used](https://www.chimerarevo.com/internet/cancellare-cache-singolo-sito-247144/).

Comment (DanielBiegler) [reply ita.json]: Cool, alright! Italian sounds so nice in my opinion, I like it very much.

**Review by DanielBiegler — CHANGES_REQUESTED** [review]
one last thing before merging, i switched to a new language code system. check the README in master for infos. So this file needs to be renamed to "IT.json" or something else if you used a specific dialect.

**Review by DanielBiegler — APPROVED** [review]

**Pull request merged**